static/openapi/prerendering-spec.json

```diff
@@ -2422,21 +2587,905 @@
                 }
               }
             }
-          },
-          "504": {
-            "description": "Gateway timeout",
-            "content": {
-              "application/json": {
-                "schema": {
-                  "type": "object",
-                  "description": "Gateway timeout",
-                  "properties": {
-                    "status": {
-                      "type": "string",
-                      "description": "Error status"
-                    },
-                    "message": {
-                      "type": "string",
+          }
+        }
+      },
+      "post": {
+        "summary": "Create prefetch configuration",
+        "tags": ["Prefetch Configurations"],
+        "description": "This endpoint creates a prefetch configuration associated with the origin.",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "cronSchedule": {
+                    "type": "string",
+                    "description": "Cron expression for scheduling"
```

Comment:
What does this expression look like?

Reply:
This is the cron expression (crontab). Something like `* * * * *`.

Reply:
Do you want me to change something here?

Reply:
This looks like output, so you are good.  I thought you were entering this data.